dvc/repo/gc.py

```diff
@@ -13,6 +13,15 @@ def _do_gc(typ, func, clist):
         logger.info("No unused '{}' cache to remove.".format(typ))
 
 
+def _raise_error_if_all_disabled(**kwargs):
+    if not any(kwargs.values()):
+        raise ValueError(
+            (
+                "Invalid Arguments. " "Either of ({}) " "needs to be enabled."
```

Comment:
What's with this separate string literals? This also needs to be `DvcException` to not be displayed as an unexpected error.

Reply:
dammit black. I'll fix it.